include/bluetooth/iso.h

```diff
@@ -577,6 +577,23 @@ int bt_iso_chan_disconnect(struct bt_iso_chan *chan);
  */
 int bt_iso_chan_send(struct bt_iso_chan *chan, struct net_buf *buf);
 
+/** @brief Configure vendor ISO data path
```

Comment:
It is of course a bit difficult to refer to since we don't expose the ISO data path command directly as an API call, but I feel like it is missing some reference to the data path ID. Suggest to add a reference to `struct bt_iso_chan_path`.

Reply:
Please elaborate, thanks.